app/store/configureStore.dev.js

```diff
@@ -0,0 +1,77 @@
+import { createStore, applyMiddleware, compose } from 'redux';
```

Comment:
This file seems suspicious. I don't think anything related to the pyodide update should need to change how our Redux stores are initialized.